### DISCUSSION TITLE
Welcome to discord-player Discussions!

### DISCUSSION THREAD

**Discussion by Androz2091** [Androz2091/discord-player#190] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by loveisglitchy** [comment]
Hey Androz, I love this package and it works amazingly! Keep up the great work man!
